### PR TITLE
Add project proposal for interactive debugging in RDataFrame

### PR DESCRIPTION
**Pull request by eguiraud**
Hi, I'd like to propose this project, I am not sure I did everything correctly for the proposal. Please let me know if I should change/improve something!

### CODE REVIEW

File: gsoc/2022/mentors.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ layout: plain
 * Mario Lassnig [mario.lassnig@cern.ch](mailto:mario.lassnig@cern.ch) CERN
 * Javier Lopez-Gomez [Javier Lopez-Gomez](mailto:j.lopez@cern.ch) CERN
 * Axel Naumann [Axel Naumann](mailto:Axel.Naumann@cern.ch) CERN
+* Enrico Guiraud [Enrico Guiraud](mailto:enrico.guiraud@cern.ch) CERN
```
Comment (hegner) [inline gsoc/2022/mentors.md:23]: could you put yourself into the alphabetical order?

File: _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Interested students please contact Enrico (see **Mentors** below) for a warm-up and evaluation task.
+
+## Requirements
+
+  * C++
+
+
+## Mentors
+
+  * **[Enrico Guiraud](mailto:enrico.guiraud@cern.ch)**
```
Comment (hegner) [inline _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md:32]: Do you have a second/ back-up mentor?

File: _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+In this project the student will add features to RDataFrame useful to inspect the state of the data at selected points of the data processing loop, enabling users to verify and inspect logical errors in their analysis code in a much more ergonomic fashion than what is currently available. The student will gain practical experience in the software development cycle of a real-world product and with the tools and best practices employed therein.
+
+## Evaluation Task
+
+Interested students please contact Enrico (see **Mentors** below) for a warm-up and evaluation task.
+
+## Requirements
+
+  * C++
```
Comment (hegner) [inline _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md:27]: that's a bit too unspecific. How much experience? any fields of application, etc?

Comment (eguiraud) [reply _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md]: i qualified it a bit more

File: gsoc/2022/mentors.md (hunk 1 above)
Comment (eguiraud) [reply gsoc/2022/mentors.md]: done

File: _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Requirements
+
+  * good knowledge of C++ and C++ templates
+
+
+## Mentors
+
+  * **[Enrico Guiraud](mailto:enrico.guiraud@cern.ch)**
+  * Enric Tejedor [etejedor@cern.ch](mailto:etejedor@cern.ch)
```
Comment (eguiraud) [inline _gsocproposals/2022/proposal_Root-interactive-debugging-rdf.md:33]:
```suggestion
  * [Enric Tejedor](mailto:etejedor@cern.ch)
```

**Pull request merged**